### PR TITLE
mongodb endpoint

### PR DESCRIPTION
**Pull request by AlvaroVega**
- Should we add PERSEO_MONGO_USER and PERSEO_MONGO_PASSWORD to env vars?

### CODE REVIEW

File: bin/perseo [hunk 1]
```diff
@@ -81,9 +82,11 @@ function loadConfiguration() {
     if (process.env.PERSEO_ENDPOINT_PORT) {
         config.endpoint.port = process.env.PERSEO_ENDPOINT_PORT;
     }
-    if (process.env.PERSEO_MONGO_HOST) {
-        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
-        config.orionDb.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
+    if (process.env.PERSEO_MONGO_ENDPOINT) {
```
Comment (fgalan) [inline bin/perseo]: The problem with this approach is that the user could use `?replicaSet=` in the PERSEO_MONGO_ENDPOINT connection string and, at the same time, a replica set in PERSEO_MONGO_RESPLICASET. If at the end we go ahead with two env vars (PERSEO_MONGO_ENDPOINT and PERSEO_MONGO_RESPLICASET vs. the option of having all them at one point in PERSEO_MONGO_ENDPOINT) at least PERSEO_MONGO_ENDPOINT should be checked to detec that `?replicaSet` is not being used.

**Comment by fgalan** [comment]
Fixes https://github.com/telefonicaid/perseo-fe/issues/243

**Comment by fgalan** [comment]
> Should we add PERSEO_MONGO_USER and PERSEO_MONGO_PASSWORD to env vars?

We could avoid adding new env vars is we "exploit" the full usage of mongo connection string at PERSEO_MONGO_ENDPOINT.

Connection string full syntax:

```
mongodb://[username:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[database][?options]]
```

File: bin/perseo (hunk 1 above)
Comment (AlvaroVega) [inline bin/perseo]: I mean PERSEO_MONGO_ENDPOINT as a list of endpoints ([host:port], not a mongodb connection string

**Comment by AlvaroVega** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM. Waiting for @cesarjorgemartinez to have a final look on this before merging

**Comment by cesarjorgemartinez** [comment]
LGTM

**Pull request merged**